lib/encrypt.py

```diff
@@ -0,0 +1,22 @@
+import sys, getopt
+from Crypto.Cipher import AES
+import binascii
+import logging
+
+rootLogger = logging.getLogger()
+
+def main(inputfile = None, outputfile = None, loglevel=logging.INFO):
```

Comment:
Long term we might as well inject the logger with a default `rootLogger = logging.getLogger()` here, for example, to avoid the global. Or find whatever the more Pythonic way to do this is. Either way, not blocking merge.